src/main/java/io/github/jsonSnapshot/SnapshotFile.java

```diff
@@ -32,18 +34,19 @@
         fileContent.append(sCurrentLine + "\n");
       }
 
-      String fileText = fileContent.toString();
+      final String fileText = fileContent.toString();
       if (StringUtils.isNotBlank(fileText)) {
-        rawSnapshots =
-            Stream.of(fileContent.toString().split(SPLIT_STRING))
-                .map(String::trim)
-                .collect(Collectors.toCollection(TreeSet::new));
```

Comment:
Retain stream-based solution if possible - they are often more expressive.

Reply:
done (sortof..let's see what you think)

Reply:
Maybe something like this would work?

```java
    try (Stream<String> lines = Files.lines(...args...)) {
        lines.filter(StringUtils::isNotBlank)
        .map(SnapshotDataItem::new)
        .forEach(storedSnapshots::add);
    } catch (IOException e) {
      createFile(this.fileName);
    }
```

Reply:
i changed to using Fiels with made the code nicer (see https://github.com/json-snapshot/json-snapshot.github.io/pull/15/commits/863d2adecadc7d005918e0d152a1a8458d2117fc#diff-c9a06f397404563d24534587bed70155R45) ,
but generally a raw snapshots string is more than one line. 

I didn't find the time figure out how to to stream the lines and then aggregate them into raw-snapshotitems. I hope this way, it's also OK